bundles/org.openhab.binding.bluetooth.govee/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,194 @@
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="temperatureWarningAlarm" type="boolean" groupName="alarms">
+				<label>Broadcast Temperature Warning</label>
+				<description>If enabled, the Govee device will notify openHAB if temperature is out of the specified range</description>
+				<default>false</default>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="temperatureWarningMin" type="decimal" min="-20" max="60" step="0.2" groupName="alarms"
+				unit="Cel">
+				<label>Min Warning Temperature</label>
+				<description>Sets the lowest acceptable temperature value before a warning should be issued</description>
+				<default>0</default>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="temperatureWarningMax" type="decimal" min="-20" max="60" step="0.2" groupName="alarms"
+				unit="Cel">
+				<label>Max Warning Temperature</label>
+				<description>Sets the highest acceptable temperature value before a warning should be issued</description>
+				<default>0</default>
+				<advanced>true</advanced>
+			</parameter>
+
+			<parameter name="humidityCalibration" type="decimal" min="-9" max="9" groupName="calibration" unit="%">
+				<required>false</required>
+				<label>Humidity Calibration</label>
+				<description>Adds offset to reported humidity</description>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="humidityWarningAlarm" type="boolean" groupName="alarms">
+				<label>Broadcast Humidity Warning</label>
+				<description>If enabled, the Govee device will notify openHAB if humidity is out of the specified range</description>
+				<default>false</default>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="humidityWarningMin" type="decimal" min="0" max="100" step="0.1" groupName="alarms"
+				unit="%">
+				<label>Min Warning Humidity</label>
+				<description>Sets the lowest acceptable humidity value before a warning should be issued</description>
+				<default>0</default>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="humidityWarningMax" type="decimal" min="0" max="100" step="0.1" groupName="alarms"
+				unit="%">
+				<label>Max Warning Humidity</label>
+				<description>Sets the highest acceptable humidity value before a warning should be issued</description>
+				<default>0</default>
+				<advanced>true</advanced>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<channel-type id="govee-temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Current Measured Temperature</label>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="govee-temperature-alarm">
+		<item-type>Switch</item-type>
+		<label>Temperature Warning Alarm</label>
+		<description>If temperature warnings are enabled, then this alarm indicates
+			whether the current temperature is out of
+			range
+		</description>
```

Comment:
Strange line break.

Reply:
blame spotless

Reply:
tried my best, but there is definitely some kinds of spotless formatting bug at play.